batch-setup/make_meta_tiles.py

```diff
@@ -433,6 +458,12 @@ def render(self, num_retries, lense):
     parser.add_argument('--allowed-missing-tiles', default=2, type=int,
                         help='The maximum number of missing metatiles allowed '
                         'to continue the build process.')
+    parser.add_argument('--use-tiles-coords-generator', type=lambda x: bool(strtobool(x)), nargs='?',
+                        const=True, default=False)
+    parser.add_argument('--coords-generator-bbox-west', type=float, help="west longitude of bounding box to build")
```

Comment:
Agree that this should be a single string arg and then a split plus assert that there are four comma-separated floats. Also if you split and assert you don't have to worry about defaults.

Reply:
changed in the newest commit